api/src/main/java/org/openmrs/module/reporting/common/Option.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.common;
+
+/**
+ * This enum is meant to represent the options which can be used to specify 
+ * matching requirements
+ */
+public enum Option {
```

Comment:
Can we rename this to something more descriptive?

Reply:
Working on this. Thank you @mseaton .